ansible/pbTestScripts/buildJDKWin.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -eu
+export TARGET_OS=windows
+export ARCHITECTURE=x64
+export JAVA_TO_BUILD=jdk8
+export VARIANT=hotspot
+export JDK7_BOOT_DIR=/cygdrive/c/openjdk/jdk7
+export JDK11_BOOT_DIR=/cygdrive/c/openjdk/jdk-11
```

Comment:
why do we exprt java 11 for an 8 build?

Reply:
Initially, it was to run Gradle- however at some point through this PR, the script figured out how to detect jdk-8, so it's been removed (though I did double check).